docs/configuration/prefill/objects_api.rst

```diff
@@ -1,59 +1,127 @@
-     from the object type contains the value.
+To configure the Objects API follow these steps:
 
-7. Click **Save**
-8. Save the form
+#. In Open Forms, navigate to: **Configuration** > **Services**
+#. Create a service for the Objects API (ORC) where the form data will be registered.
 
-The Objects API configuration is now complete.
+   a. Click **Service toevoegen**.
+   b. Fill out the form:
 
-.. note::
+      * **Label**: *Fill in a human readable label*, for example: ``My Objects API``
+      * **Type**: Select the type: ``ORC``
+      * **API root url**: The root of this API, *for example* ``https://example.com/objecten/api/v2/``
+
+      * **Authorization type**: Select the option: ``API Key``
+      * **Header key**: Fill in ``Authorization``
+      * **Header value**: Fill in ``Token <tokenValue>`` where ``<tokenValue>`` is replaced by the token provided by the backend service
+      * **OAS**: URL that points to the OAS, same URL as used for **API root url** with ``/schema/openapi.yaml`` added to it
+        *for example:* ``https://example.com/objecten/api/v1/schema/openapi.yaml``
+
+      * **NLX**: Support for NLX can be selected here if enabled in the installation
+
+   c. Click **Opslaan** and repeat to create configuration for the other component.
+
+#. Create a service for the Objecttypes API (ORC).
+
+   a. Click **Service toevoegen**.
+   b. Fill out the form:
+
+      * **Label**: *Fill in a human readable label*, for example: ``My Objecttypes API``
+      * **Type**: Select the type: ``ORC``
+      * **API root url**: The root of this API, *for example* ``https://example.com/objecttypen/api/v2/``
+
+      * **Authorization type**: Select the option: ``API Key``
+      * **Header key**: Fill in ``Authorization``
+      * **Header value**: Fill in ``Token <tokenValue>`` where ``<tokenValue>`` is replaced by the token provided by the backend service
+      * **OAS**: URL that points to the OAS, same URL as used for **API root url** with ``/schema/openapi.yaml`` added to it
+        *for example:* ``https://example.com/objecttypen/api/v1/schema/openapi.yaml``
 
-   The Objects API prefill can be used not only for prefilling and showing the data to the user,
-   but it can also work together with the Objects API registration plugin
-   (see :ref:`Objects API configuration (English) <configuration_registration_objects>`) in order to make 
-   modifications to an existing object in the Objects API. This requires the form user to be the "owner"
-   of the object and the necessary configuration in the Objects API registration at the form level
-   (by choosing to update the existing object).
+      * **NLX**: Support for NLX can be selected here if enabled in the installation
 
-   .. image:: _assets/update_existing_object.png
+   c. Click **Opslaan** and repeat to create configuration for the other component.
+
+#. Navigate to **Configuration** > **Overview**. In the **Prefill plugins** group,
```

Comment:
I'm updating the docs. And yeah, the menu is crazy, ideally we'd fix this with a proper custom admin UI where we can build a nice configuration page so that we don't need individual menu entries.